README.md

```diff
@@ -17,7 +33,13 @@ This application is also responsible for sending statistics to the Performance P
 
 This is useful for debugging and populating last_login of the user.
 
-### Statistics sent over to the performance platform
+### Sinatra routes
+
+- `POST /healthcheck` - AWS ELB target group health checking
+- `POST /logging/post-auth/user/?:username?/mac/?:mac?/ap/?:called_station_id?/site/?:site_ip_address?/result/:authentication_result` - Persist a
```

Comment:
Oh sorry this is:

GET /healthcheck
POST /logging/post-auth

params:
  :username
  :mac
  :called_station_id
  :site_ip_address
  :authentication_result